autoload/lightline/delphinus/components.vim

```diff
@@ -10,32 +10,30 @@ scriptencoding utf-8
 if g:lightline_delphinus_use_nerd_fonts_glyphs
   let s:mo_glyph = ''
   let s:help_glyph = ''
+  let s:ro_glyph = ''
+elseif g:lightline_delphinus_use_powerline_glyphs
+  let s:ro_glyph = ''
```

Comment:
The both option `g:lightline_delphinus_use_powerline_glyphs` & `g:lightline_delphinus_use_nerd_fonts_glyphs` are *non-exclusive*.  The former controls glyphs in powerline-fonts such as `>`, `<` separators, and the “lock” icon “🔒”.  The latter controls the other icons.

So divide this `if` to choose them separately.